config/prometheus/collector_alerts.yaml

```diff
@@ -98,15 +88,16 @@ spec:
         severity: Warning
   - name: logging_clusterlogging_telemetry.rules
     rules:
-    - expr: |
```

Comment:
same comment as previous...is this metric only available from fluentd metrics?

Reply:
yes